lti_consumer/api.py

```diff
@@ -36,12 +36,12 @@ def _get_or_create_local_lti_config(lti_version, block_location):
     return lti_config
 
 
-def get_lti_consumer(config_id=None, block=None):
+def _get_lti_config(config_id=None, block=None):
```

Comment:
Why are we defaulting to ``None`` for ``config_id`` and ``block``? Is there any use case for that?

Reply:
@shimulch Because we can retrieve a configuration either using the `block` or the `config_id`, and there's no need to use both.

Eventually, we'll only use `config_id` but there's no full model support for it now, hence the current implementation.
I've just added an extra method to abstract re-usable code for the deep linking implementation.

> Is there any use case for that?

Yes, LTI-AGS and other advantage APIs rely on this feature.